packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -426,6 +446,16 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
         tooltip: localizations.nextPageTooltip,
         onPressed: (!_rowCountApproximate && (_firstRowIndex + widget.rowsPerPage >= _rowCount)) ? null : _handleNext,
       ),
+      if (widget.showFirstLastButtons)
+        IconButton(
+          icon: const Icon(Icons.skip_next),
+          padding: EdgeInsets.zero,
+          tooltip: localizations.lastPageTooltip,
+          onPressed: (!_rowCountApproximate &&
+              (_firstRowIndex + widget.rowsPerPage >= _rowCount))
```

Comment:
I'd break the test condition out into a one-line arrow function and replace this and the usage at 447

Reply:
Sounds good, done